src/impy/models/qgsjet.py

```diff
@@ -81,7 +81,12 @@ def __init__(self, event_kinematics, seed=None, logfname=None):
         super().__init__(seed, logfname)
 
         info(5, "Initializing QGSJET-II")
+        _cached_data_dir(
+            "https://github.com/impy-project/impy/releases/download"
+            "/zipped_data_v1.0/qgsjet_v001.zip"
+        )
         datdir = path.join(base_path, impy_config["qgsjet"]["datdir"])
```

Comment:
Same here, get directory from the _Chached_data_dir. Also, no os.path anymore.

Reply:
Done.